### DISCUSSION TITLE
Azure DevOps Git & OAuth 2.0

### DISCUSSION THREAD

**Discussion by stevendick-work** [fluxcd/flux2#1824] [Proposals]
Microsoft have documented experimental support for using OAuth 2.0 instead of PATs (Personal Access Tokens) [here](https://github.com/microsoft/Git-Credential-Manager-Core/blob/main/docs/azrepos-users-and-tokens.md).

Is this something on Flux's roadmap?

**Comment by kingdonb** [comment]
We are talking about this at Bug Scrub today. We have questions about how the refresh token works and what it means when it is "actively used" – whether there would be issues refreshing the token if you don't have any git commits for a period of time etc.

We do not have this on the roadmap at present and haven't really thought about how it could be implemented!
